src/site/_includes/moduk/components/notification-banner/success.njk

```diff
@@ -0,0 +1,13 @@
+{% from "moduk/components/notification-banner/macro.njk" import modukNotificationBanner -%}
```

Comment:
Is this needed – seems identical to the existing success example in moduk-frontend?

Reply:
I didn't see it  let me look again

Reply:
Ah yes ok, must have missed it